clean-package.config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "remove": ["engines", "lint-staged", "jest", "eslintConfig", "scripts"]
```

Comment:
We want to undo fields other than engines?

Reply:
All fields are development config dont make sense to be in the published package